File: Java/PrimeiroCodigo/src/classe/DataTeste.java
package classe;

import java.util.Scanner;

public class DataTeste
{
    public static void main(String[] args)
    {
        Scanner getUserString = new Scanner(System.in);
        
        Data calendario1 = new Data();
        System.out.print("\nCalendario 1: \nDigite o dia:");
        int dia = Integer.parseInt(getUserString.next());
        calendario1.dia = dia;
        System.out.print("Digite o mês:");
        int mes = Integer.parseInt(getUserString.next());
        calendario1.mes = mes;
        System.out.print("Digite o ano:");
        int ano = Integer.parseInt(getUserString.next());
        calendario1.ano = ano;
        
        Data calendario2 = new Data();
        calendario2.dia =6;

        Data calendario3 = new Data(05, 22, 2025);


        System.out.printf(calendario1.getDateFormated(1));
        System.out.printf(calendario2.getDateFormated(2));
        System.out.printf(calendario3.getDateFormated(3));

        String n = Integer.toString(calendario1.dia);
        System.out.println(n);

        getUserString.close();
    }
}

File: Python/python Java/Classe/DataTeste.py
from Data import Data1
calendario1 = Data1()
calendario2 = Data1()

print("\nCalendario 1: \nDigite o dia:")
dia = int(input())
calendario1.dia = dia
print("Digite o mês:")
mes = int(input())
calendario1.mes = mes
print("Digite o ano:")
ano = int(input())
calendario1.ano = ano

print("\nCalendario 2: \nDigite o dia:")
dia = int(input())
calendario2.dia = dia
print("Digite o mês:")
mes = int(input())
calendario2.mes = mes
print("Digite o ano:")
ano = int(input())
calendario2.ano = ano

print(f"\n\nCalendario 1: \nDia: {calendario1.dia} \nMês: {calendario1.mes} \nAno: {calendario1.ano}")
print(f"\n\nCalendario 2: \nDia: {calendario2.dia} \nMês: {calendario2.mes} \nAno: {calendario2.ano}")
